test/sagemaker_tests/tensorflow/tensorflow2_training/resources/hvdbasic/train_hvd_env_vars.py

```diff
@@ -0,0 +1,19 @@
+import json
+import os
+import horovod.tensorflow as hvd
+
+hvd.init()
+
+with open('/opt/ml/model/local-rank-%s-rank-%s' % (hvd.local_rank(), hvd.rank()), 'w+') as f:
+    basic_info = {'local-rank': hvd.local_rank(), 'rank': hvd.rank(), 'size': hvd.size()}
+
+    print(basic_info)
+    json.dump(basic_info, f)
+
+val = os.environ.get('AWS_CONTAINER_CREDENTIALS_RELATIVE_URI')
+host = os.environ.get('SM_CURRENT_HOST')
+
+assert val is not None
+assert host is not None
```

Comment:
nit: is there a scenario where val or host will be None? os.environ.get should throw an exception if the env variable is not found

Reply:
`os.environ.get` will not throw an exception if the environment variable is not found; it defaults to `None`: https://docs.python.org/3/library/stdtypes.html#dict.get

If you attempt to access the environment variable directly (`os.environ["NONEXISTENT_VAR"]`), it will raise a [`KeyError`](https://docs.python.org/3/library/exceptions.html#KeyError).